ql/src/test/results/clientpositive/llap/acid_table_directories_test.q.out

```diff
@@ -163,13 +170,6 @@ POSTHOOK: Input: default@acidparttbl@p=200
 ### ACID DELTA DIR ###
 ### ACID DELTA DIR ###
 ### ACID DELTA DIR ###
-#### A masked pattern was here ####
```

Comment:
in case of a hive patch, usually, I don't care about result ordering change, however, this is a hadoop upgrade, this is not expected, can we explain this change?

Reply:
[HADOOP-12502](https://issues.apache.org/jira/browse/HADOOP-12502) is the culprit and earlier listing was sorted now it isn't so the Ls -R output will change intermittently so we can't have this test only hence disabled